torch/_dynamo/symbolic_convert.py

```diff
@@ -684,18 +688,10 @@ def step(self):
             getattr(self, inst.opname)(inst)
 
             return inst.opname != "RETURN_VALUE"
-        except BackendCompilerFailed:
-            raise
-        except Unsupported as exc:
-            exc.real_stack.append(self.frame_summary())
+        except Unsupported:
             if self.empty_checkpoint():
                 raise
             log.debug("step triggered compile", exc_info=True)
-        except Exception as exc:
-            real_stack = getattr(exc, "real_stack", [])
-            real_stack.append(self.frame_summary())
-            exc.real_stack = real_stack  # type: ignore[attr-defined]
-            raise
```

Comment:
It's not necessary to append stack frames onto exceptions as they bubble up. Even better, it is not necessary to attach stacks either; TracingContext takes care of this.